test/fixtures/comments.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  image: one
+  user: one
+  text: First post!!!1!
+
+two:
+  image: two
+  user: one
+  text: I like comments
```

Comment:
Added one more. Once we start sorting we can add more specifically for those test cases.